README.md

```diff
@@ -1 +1,37 @@
-# training-hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- Please make sure to add the `project name` and delete this line. Write `Hello Microverse` for example instead of `Project Name`, to have a descriptive project name :spiral_notepad:
-  Also kindly add a picture of the project, to help people to have an idea about your project and delete the app_screenshot.png image in your project.

Reply:
@cynthiainga project name updated and the particular line is deleted. The picture is also deleted and a screenshot is provided.